maxim/logger/writer.py

```
@@ -366,24 +405,79 @@ def flush_log_files(self):
+                        continue
                     scribe().warning(
-                        f"[MaximSDK] Failed to access filesystem. Error: {e}"
+                        "[MaximSDK] Failed to push spilled log file %s "
+                        "(attempt %s/%s). Error: %s",
+                        file,
+                        attempts,
+                        MAX_FILE_REPLAY_ATTEMPTS,
+                        e,
                     )
                     if self.raise_exceptions:
                         raise Exception(e)
```

Ruff (0.15.21)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)